src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
+type Color = 'red' | 'blue' | 'green';
+type Shape = 'circle' | 'rectangle' | 'triangle';
```

Comment:
You use Shape values in several places in your code, so it makes sense to make it `Enum`, not just `type`.
Use `Enum` here instead